packages/sdk/src/reads/convertCrvToCvxWithCvxMining.test.ts

```diff
@@ -0,0 +1,17 @@
+import { publicClientMainnet } from "../../tests/globals.js";
+import { convertCrvToCvxWithCvxMining } from "./convertCrvToCvxWithCvxMining.js";
+import { assert, expect, test } from "vitest";
+
+test("convert crv to cvx with cvx mining should work correctly", async () => {
+  const cvxMining = "0x3c75BFe6FbfDa3A94E7E7E8c2216AFc684dE5343" as const;
+  const amount = 1000n as const;
+
+  const result = await convertCrvToCvxWithCvxMining(publicClientMainnet, {
+    cvxMining,
+    amount,
+  });
+
+  assert(result !== undefined && result !== null);
```

Comment:
this line is not needed, if we check that result is type of bigint then we know that this is not null and undefined